packages/shared/package.json

```diff
@@ -24,17 +24,17 @@
     "swr": "^1.3.0",
     "tsup": "^6.2.3",
     "typescript": "*",
-    "vitest": "^0.23.4"
-  },
-  "peerDependencies": {
+    "vitest": "^0.23.4",
     "@testing-library/dom": "^8.19.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
     "react-test-renderer": "^18.2.0"
   },
+  "peerDependencies": {
```

Comment:
In order for this change to work we need to make sure that local npm is >= 7 and installed peers dependencies by default. Can you please verify that everything is working fine for npm too?